Doc/library/sqlite3.rst

```diff
@@ -1535,6 +1466,103 @@ the context manager is a no-op.
 .. literalinclude:: ../includes/sqlite3/ctx_manager.py
 
 
+.. _sqlite3-concepts:
+
+Concepts
+--------
+
+.. _sqlite3-types:
+
+SQLite and Python types
+~~~~~~~~~~~~~~~~~~~~~~~
+
+SQLite natively supports the following types: ``NULL``, ``INTEGER``,
+``REAL``, ``TEXT``, ``BLOB``.
+
+The following Python types can thus be sent to SQLite without any problem:
+
++-------------------------------+-------------+
+| Python type                   | SQLite type |
++===============================+=============+
+| :const:`None`                 | ``NULL``    |
++-------------------------------+-------------+
+| :class:`int`                  | ``INTEGER`` |
++-------------------------------+-------------+
+| :class:`float`                | ``REAL``    |
++-------------------------------+-------------+
+| :class:`str`                  | ``TEXT``    |
++-------------------------------+-------------+
+| :class:`bytes`                | ``BLOB``    |
++-------------------------------+-------------+
+
+
+This is how SQLite types are converted to Python types by default:
+
++-------------+----------------------------------------------+
+| SQLite type | Python type                                  |
++=============+==============================================+
+| ``NULL``    | :const:`None`                                |
++-------------+----------------------------------------------+
+| ``INTEGER`` | :class:`int`                                 |
++-------------+----------------------------------------------+
+| ``REAL``    | :class:`float`                               |
++-------------+----------------------------------------------+
+| ``TEXT``    | depends on :attr:`~Connection.text_factory`, |
+|             | :class:`str` by default                      |
++-------------+----------------------------------------------+
+| ``BLOB``    | :class:`bytes`                               |
++-------------+----------------------------------------------+
+
+The type system of the :mod:`sqlite3` module is extensible in two ways: you can
+store additional Python types in an SQLite database via
+:ref:`object adapters <sqlite3-adapters>`,
+and you can let the :mod:`sqlite3` module convert SQLite types to
+Python types via :ref:`converters <sqlite3-converters>`.
+
+
+.. _sqlite3-controlling-transactions:
+
+Controlling Transactions
+~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
Transaction Control
~~~~~~~~~~~~~~~~~~~
```

Concept-oriented rather than task-oriented, reflecting both the Diataxis content type and the actual focus of the section